src/main/scala/cognite/spark/v1/CDFSparkException.scala

```diff
@@ -0,0 +1,27 @@
+package cognite.spark.v1
+
+trait CDFSparkException {}
```

Comment:
We capitalize this as `CdfSparkException` (change the file name as well), but apart from that this, looks pretty good to me!